Makefile

```diff
@@ -13,7 +13,6 @@ reset-db:
 docker:
 	scripts/get_cogctl.sh
 	docker build \
-		   --build-arg MIX_ENV=prod \
 		   -t $(DOCKER_IMAGE) .
```

Comment:
Totally just a minor style thing, but this might be easier to read on a single line.